client/components/plans/plan-icon/style.scss

```diff
@@ -2,21 +2,39 @@
 	width: inherit;
 	height: inherit;
 	background-repeat: no-repeat;
+}
 
-	&.is-free-plan {
-		background-image: url('/calypso/images/plans/plan-beginner.svg');
-	}
-
-	&.is-personal-plan {
-		background-image: url('/calypso/images/plans/plan-personal.svg');
-	}
+.plan-icon__free {
+	.st0 { fill:#78DCFA; }
+	.st1 { fill:#FFFFFF; }
+	.st2 { fill:#485567; }
+	.st3 { fill:#677B97; }
+	.st4 { fill:#7C94B6; }
+}
 
-	&.is-premium-plan {
-		background-image: url('/calypso/images/plans/plan-premium.svg');
-	}
+.plan-icon__personal {
+	.st0 { fill:#F0B849; }
```

Comment:
Yes, absolutely. I copied the fills from the minified svg and didn't hit that yet.